### ISSUE TITLE
Drop down lists on desktop stop scrolling for profiles and tags

### ISSUE THREAD

**Issue by smashopen** [smashopen/smash#26]
If the list is too long. Browser scrolls instead of list. Keyboard navigation doesn't work anymore. Browser, bootstrap or app issue? Investigate
